Orange/widgets/evaluate/owtestlearners.py

```diff
@@ -166,6 +166,7 @@ class OWTestLearners(OWWidget):
 
     class Error(OWWidget.Error):
         class_required = Msg("Train data input requires a class variable")
+        too_many_classes = Msg("Too many target variables.")
```

Comment:
Sorry. Misread. Not enough sleep. Too much lecturing. Coffee too schwach. All of them.